ontology/uco-gufo.ttl

```diff
@@ -48,6 +58,21 @@ drafting:Quality
 	rdfs:seeAlso <https://github.com/ucoProject/UCO/issues/535> ;
 	.
 
+drafting:Vehicle
+	a owl:Class ;
+	rdfs:subClassOf uco-observable:Device ;
+	rdfs:seeAlso <https://github.com/ucoProject/UCO/issues/567> ;
+	.
+
+drafting:Velocity
+	a
+		gufo:Kind ,
```

Comment:
Like a Bearing, the Velocity is an Intrinsic Aspect ("an aspect that depends on a single concrete individual in which it inheres"). Velocity can be measured, hence it is a `Quality`:
`:Velocity rdfs:subClassOf gufo:Quality ;`

And because a `Quality` inheres in a concrete individual, we should add that as well:

`    gufo:inheresIn :Vehicle .`

Reply:
Ah, I see the inherence you mean now.  I'll add some `owl:Restriction`s.

Reply:
`owl:Restriction`s added.

We agree on bearing and velocity being `Quality`s.  But, I think we're disagreeing on whether bearing and velocity are rigid.  Are you arguing that bearing and velocity are semi-rigid instead?

My argument that these **are** rigid: Just because a person is sitting still during a meeting doesn't mean that person't doesn't have a velocity.  They **do** have a velocity, of 0 miles per hour, and often there may be some device in a pocket or on a wrist to attest to that.

Reply:
The persons, as individual beings, are `Object`s. Things like velocity, weight or length are individual beings too, but as opposed to `Object`s they express reified `Aspect`s during their bearer's existence. 

Now, the universals that the individual `Object`s instantiate - Endurants, depicted as the blue column below - can show some form of rigidness and sortal behaviour, and are categorised (typed) according to that; The universal Person can thus be typed as `gufo:Kind`s. 

The universals that the individual `Aspect`s instantiate cannot show rigidness nor sortality; because, they are not substantial things but properties. I'm not sure why gufo categorises the `Aspect` as Endurant; I consider them separate from it and equivalent to the green Quality / Property column in the picture below. I'll definitely ask @tgoprince about it.



![OntoSquareAndSextet](https://github.com/ucoProject/UCO-Profile-gufo/assets/8479553/f052886a-c186-425a-a1c0-b20ce4f21f93)

Reply:
In line with the resolution of the previous issue, we can close this conversation n the same footing, i.e., accept the safer option 2 to leave line 69 as it is, and hope for a guiding response from the gUFO-folks on this.